commerce_coordinator/apps/commercetools/catalog_info/utils.py

```diff
@@ -103,6 +106,10 @@ def typed_money_to_string(money: CTTypedMoney) -> str:
     cur = Currency(money.currency_code)
 
     def _format(cost):
+        if money_as_decimal:  # pragma no cover
+            # This is used to support legacy ecomm mfe
+            return str(cost)
```

Comment:
nit: The variable name reads as money in decimal type. But we are type-casting it to a string here.